to_backend_theme/static/src/scss/style.scss

```diff
@@ -100,12 +100,12 @@ body {
 	}
 }
 
-@media (min-width: 768px) {
+@media (min-width: 717px) {
```

Comment:
Sao mình cần thay đổi min width về kích thước không tiêu chuẩn?

Reply:
Vâng, em thử với galaxy fold thì chỗ view list các ô chọn to quá nên em giảm min-width xuống ạ. @davidtranhp 

![Screenshot from 2022-04-22 15-20-36](https://user-images.githubusercontent.com/90305443/164648416-faf89115-a156-425b-8d33-83711211b5f9.png)

Reply:
Nên dùng `@media (max-width: 767.98px)` thì hơn

Reply:
Dạ vâng anh